src/components/Chip/index.tsx

```diff
@@ -0,0 +1,32 @@
+import { ReactNode } from 'react';
+
+import { type VariantProps, cva } from 'class-variance-authority';
+
+const ChipCSS = cva(
+  'flex w-fit items-center rounded-2xl border border-primary text-primary',
+  {
+    variants: {
+      variant: {
+        small: ' px-2 py-0.5 text-[10px]',
+        large: ' px-4 py-2',
```

Comment:
아주 사소한 부분이지만 공백이 들어갔네요
```suggestion
        small: 'px-2 py-0.5 text-[10px]',
        large: 'px-4 py-2',
```

Reply:
실수했네요.. 감사합니다